api.go

```diff
@@ -43,13 +43,16 @@ func init() {
 	if Config.BUILDLOCATION == "" {
 		logging.LogToFile("Missing environment variable BUILD_LOCATION")
 		panic("Missing environment variable BUILD_LOCATION")
-
 	}
 	Config.FRANKLINAPIURL = os.Getenv("API_URL")
 	if Config.FRANKLINAPIURL == "" {
 		logging.LogToFile("Missing environment variable API_URL")
 		panic("Missing environment variable API_URL")
-
+	}
+	Config.DEPLOYROOTPATH = os.Getenv("DEPLOY_ROOT_FOLDER")
```

Comment:
There is a config struct above you need to add `DEPLOYROOTPATH` to that as well.